package.json

```diff
@@ -70,53 +71,10 @@
     "extend-error": "git://github.com/ianwremmel/extend-error.git",
     "gm": "~1.18.1",
     "grunt-file-exists": "^0.1.4",
-    "lodash.assign": "^3.1.0",
```

Comment:
These need to stick around for the legacy sdk, but I'm very excited to get rid of them once the web client updates